src/ServicePulse.Host/vue/src/composables/isRouteSelected.ts

```diff
@@ -11,5 +11,6 @@ export default function isRouteSelected(path: string) {
     return true;
   }
 
-  return useLink({ to: path }).route.value.name === route.name;
+  const pathRoute = useLink({ to: path }).route.value;
+  return route.matched.some((match) => match.name === pathRoute.name);
```

Comment:
So "matched" should contain the parent and the child route correct?
And if so, then do we still need the ors in for example https://github.com/Particular/ServicePulse/pull/1748/files#diff-c6d4b62c099bf184d1bfc0cbe5fa16dcad0c1039acaf718429ca4e8b3d0030b3R20

Reply:
they're different routes (one isn't an ancestor of the other)